File: Python/model/Rezultat.py
class Rezultat:
    def __init__(self):
        self._prvi = 0
        self._drugi = 0
        self._treci = 0

    def is_pocetak(self):
        return self._prvi == 0 and self._drugi == 0 and self._treci == 0

    @property
    def prvi(self):
        return self._prvi

    @prvi.setter
    def prvi(self, prvi):
        self._prvi = prvi

    @property
    def drugi(self):
        return self._drugi

    @drugi.setter
    def drugi(self, drugi):
        self._drugi = drugi

    @property
    def treci(self):
        return self._treci

    @treci.setter
    def treci(self, treci):
        self._treci = treci

File: PHP/src/Jakopec/Rezultat.php
<?php


namespace Jakopec;


class Rezultat
{

    private $prvi;
    private $drugi;
    private $treci;

    /**
     * Rezultat constructor.
     * @param int $prvi
     * @param int $drugi
     * @param int $treci
     */
    public function __construct(int $prvi = 0, int $drugi = 0, int $treci = 0)
    {
        $this->prvi = $prvi;
        $this->drugi = $drugi;
        $this->treci = $treci;
    }

    /**
     * @return int
     */
    public function getPrvi(): int
    {
        return $this->prvi;
    }

    /**
     * @param int $prvi
     */
    public function setPrvi(int $prvi)
    {
        $this->prvi = $prvi;
    }

    /**
     * @return int
     */
    public function getDrugi(): int
    {
        return $this->drugi;
    }

    /**
     * @param int $drugi
     */
    public function setDrugi(int $drugi)
    {
        $this->drugi = $drugi;
    }

    /**
     * @return int
     */
    public function getTreci(): int
    {
        return $this->treci;
    }

    /**
     * @param int $treci
     */
    public function setTreci(int $treci)
    {
        $this->treci = $treci;
    }

    public function isPocetak(): bool
    {
        return $this->prvi == 0 && $this->drugi == 0 && $this->treci == 0;
    }

}